doc/source/conf.py

```diff
@@ -22,13 +22,19 @@
 import os
 import pathlib
 import re
+from datetime import datetime
 
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'Apache Arrow Flight SQL adapter for PostgreSQL'
-copyright = '2022-2023, Apache Arrow Developers'
-author = 'Apache Arrow Developers'
+project = 'Apache Arrow Flight SQL Adapter for PostgreSQL'
```

Comment:
"adapter" is the project name. Can we use "adapter" literally here?

```suggestion
project = 'Apache Arrow Flight SQL adapter for PostgreSQL'
```